src/screens/Inventory/components/InventoryInfo.tsx

```diff
@@ -64,17 +69,16 @@ function InventoryInfo(props: InventoryInfoProps) {
           <Typography variant="h2" color="textPrimary">
             {product.name}
           </Typography>
-          <Typography variant="caption">Last Updated</Typography>
-          <Typography variant="caption">{lastUpdated}</Typography>
+          <Typography variant="caption">{intl(words.last_updated_date, lastUpdated)}</Typography>
```

Comment:
I'm not sure what formatting `.last_updated_date` is using but it doesn't seem right in English or Burmese. We intentionally made the date formatting location-sensitive, so that could probably be excluded from the translation.
![image](https://user-images.githubusercontent.com/21160510/115499162-cce44880-a223-11eb-9d79-c07d8f6468ab.png)